webapp/app/templates/content/landing_page.html

```diff
@@ -59,6 +59,13 @@
             ],
             'list-items': None,
         },
+        'reference': {
+          'heading': _('info-stl.reference.heading'),
+          'paragraphs': [
+              (_('info-stl.reference.p1-content', link=url_for('index'))),
+          ],
+          'list-items': None,
+      },
```

Comment:
Looks like that the } is not in the correct column

Reply:
thanks